server/integration/testsuite/src/test/java/org/infinispan/server/test/query/RemoteQueryStringIT.java

```diff
@@ -67,11 +59,10 @@
    @BeforeClass
    public static void before() {
       JavaArchive programmaticSearchMappingProviderArchive = ShrinkWrap.create(JavaArchive.class)
-            .addClass(TestAnalyzerProvider.class)
             .addClass(TestSearchMappingFactory.class)
             .addClass(TestSearchMappingFactory.MySearchableEntity.class)
             .add(new StringAsset("Dependencies: org.infinispan.query, org.hibernate.search.engine"), "META-INF/MANIFEST.MF")
-            .addAsServiceProvider(ProgrammaticSearchMappingProvider.class, TestAnalyzerProvider.class);
```

Comment:
This was also used to validate that TestAnalyzerProvider is indeed loadable in the very complex env of the server. I'd like to keep that test, even though now all those analyzers are preent by default. Let's still add one like that 'standard-with-stop' used in embedded test.

Reply:
The protobuf used in this test comes from the protostream codebase and hardcodes "standard", so this test needs to be rewritten, would you mind doing it later?

Reply:
No. So we need to change bank.proto model?

Reply:
I think it's better to leave it as is, as it's used throughout the code. I'd prefer to rewrite this test using more weird analyzers and different analyzers per field.

Reply:
A good test would be to actually deploy a new analyzer like http://www.atilika.org/, and use it on one of the fields.

Reply:
https://issues.jboss.org/browse/ISPN-9433